skills/dev-figma-v3-migration-plugin/references/guide.md

```
@@ -0,0 +1,372 @@
+- 개요/구조: `## 개요`, `### 디렉토리 구조`
+- 운영 절차: `## 환경 설정`, `## 매핑 최신화 프로세스`
+- 구현 패턴: `## 매핑 파일 작성 가이드`, `## 타입 시스템`
+- 장애 대응: `## 트러블슈팅`
+
+## 개요
+
+Figma V3 Migration 플러그인은 SEED Design System V2 컴포넌트를 V3로 마이그레이션하는 Figma 플러그인입니다.
+
+### 디렉토리 구조
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)